api/src/main/java/org/openmrs/Tag.java

```diff
@@ -0,0 +1,80 @@
+/**
+ * This Source Code Form is subject to the terms of the Mozilla Public License,
+ * v. 2.0. If a copy of the MPL was not distributed with this file, You can
+ * obtain one at http://mozilla.org/MPL/2.0/. OpenMRS is also distributed under
+ * the terms of the Healthcare Disclaimer located at http://openmrs.org/license.
+ *
+ * Copyright (C) OpenMRS Inc. OpenMRS is a registered trademark and the OpenMRS
+ * graphic logo is a trademark of OpenMRS Inc.
+ */
+package org.openmrs;
+
+import org.openmrs.BaseOpenmrsData;
+import org.openmrs.User;
+import java.io.Serializable;
+import java.util.Date;
+import java.util.UUID;
+
+/**
+ * Please note that a corresponding table schema has been created in liquibase.xml.
+ */
+public class Tag extends BaseOpenmrsData implements Serializable {
+	
+	private Integer id;
+	
+	private String tag;
+	
+	private String objectUuid;
+	
+	private String objectType;
+	
+	private User creator;
+	
+	private Date dateCreated;
+	
```

Comment:
I thought `creator` and `dateCreated` are already defined in one of the super classes